Gruntfile.js

```diff
@@ -132,8 +132,8 @@ module.exports = grunt => {
                         'not ie <= 12',  // all versions -- edge is separate
                     ],
                 },
-                useBuiltIns: 'usage',
-                corejs: 3,
+                // useBuiltIns: 'usage',
+                // corejs: 3,
```

Comment:
Everything compiled file except babel/webpack were complaining about core-js not being resolved correctly by the vexflow library. VexFlow doesn't use core-js, and I couldn't find any direct imports of core-js in Music21j.

Reply:
see below -- it is used by music21j, and might be a bug in vexflow build if it doesn't work in this system.

Reply:
and at least one system that uses music21j also uses core-js so if having Vexflow in the system means that core-js can't be used anywhere upstream then that would be a problem.

Reply:
When I put core-js back in, it works perfectly for me.  Maybe I'm missing something in the dependencies list that I installed manually?  What's the error you are seeing?